### ISSUE TITLE
cupyx.scipy.fft functions not found in release

### ISSUE THREAD

**Issue by HeeebsInc** [cupy/cupy#6681]
### Description

Looking at the [documentation](https://docs.cupy.dev/en/latest/reference/generated/cupyx.scipy.fft.dct.html), it shows that cupyx.scipy.fft.dct is in the latest release but I cannot find it in any pip wheels.  I tried downloading through conda and PyPi and could not find it.  

Can you please direct me toward a repo/version that contains these functions? 

### Idea or request for content

Ensure that a function documented is actually in the release.  I know there is a ton of stuff to keep track since this repo covers so much so this was bound to happen.

**Comment by leofang** [comment]
The new FFT functions like `dct` are just added recently, and currently they only exist in the master branch (and IIUC the pre-release wheels starting v11.0.0a1). So to use `dct` you need to install CuPy either from source or from the pre-release wheel, see the instruction [here](https://docs.cupy.dev/en/stable/install.html#installing-cupy-from-pypi). 

If you are reading the "latest" docs (the green toggle on the bottom right of the page) it reflects the status in the master branch. If you switch to the docs of any other versions it'd be gone.

**Comment by emcastillo** [comment]
```
Use pip install cupy-cudaXXX -f https://pip.cupy.dev/pre to install pre-release (development) versions.
```

**Comment by HeeebsInc** [comment]
Thank you guys!  I was able to download it successfully.  
The only issue now is that I am facing an import error when including cupy.  It works with the stable released versions.  I have cudatoolkit=11.3 installed with conda and used cupy-cuda113

below is the output

ImportError: 
================================================================
Failed to import CuPy.

If you installed CuPy via wheels (cupy-cudaXXX or cupy-rocm-X-X), make sure that the package matches with the version of CUDA or ROCm installed.

On Linux, you may need to set LD_LIBRARY_PATH environment variable depending on how you installed CUDA/ROCm.
On Windows, try setting CUDA_PATH environment variable.

Check the Installation Guide for details:
  https://docs.cupy.dev/en/latest/install.html

Original error:
  ModuleNotFoundError: No module named 'fastrlock'



I checked my package list and it shows I have fastrlock installed - see below.   

I appreciate your help!  Cupy has accelerated our pipeline by 10x - your work is incredible

# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
argon2-cffi               21.3.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
asttokens                 2.0.5              pyhd3eb1b0_0  
attrs                     21.4.0                   pypi_0    pypi
backcall                  0.2.0              pyhd3eb1b0_0  
backoff                   1.10.0                   pypi_0    pypi
beautifulsoup4            4.11.1                   pypi_0    pypi
blas                      1.0                         mkl  
bleach                    5.0.0                    pypi_0    pypi
ca-certificates           2022.3.29            h06a4308_1  
cachetools                5.0.0                    pypi_0    pypi
certifi                   2021.10.8                pypi_0    pypi
cffi                      1.15.0                   pypi_0    pypi
charset-normalizer        2.0.12                   pypi_0    pypi
colorama                  0.4.4                    pypi_0    pypi
colorlog                  6.6.0                    pypi_0    pypi
colormap                  1.0.3                    pypi_0    pypi
cudatoolkit               11.3.1               h2bc3f7f_2  
cupy-cuda113              10.3.1                   pypi_0    pypi
cycler                    0.11.0                   pypi_0    pypi
debugpy                   1.5.1            py38h295c915_0  
decorator                 5.1.1              pyhd3eb1b0_0  
defusedxml                0.7.1                    pypi_0    pypi
easydev                   0.10.1                   pypi_0    pypi
entrypoints               0.4              py38h06a4308_0  
et-xmlfile                1.1.0                    pypi_0    pypi
executing                 0.8.3              pyhd3eb1b0_0  
fastjsonschema            2.15.3                   pypi_0    pypi
fastrlock                 0.8                      pypi_0    pypi
google-api-core           2.7.2                    pypi_0    pypi
google-api-python-client  2.36.0                   pypi_0    pypi
google-auth               2.6.6                    pypi_0    pypi
google-auth-httplib2      0.1.0                    pypi_0    pypi
google-auth-oauthlib      0.5.1                    pypi_0    pypi
google-cloud-core         2.3.0                    pypi_0    pypi
google-cloud-storage      1.43.0                   pypi_0    pypi
google-crc32c             1.3.0                    pypi_0    pypi
google-resumable-media    2.3.2                    pypi_0    pypi
googleapis-common-protos  1.56.0                   pypi_0    pypi
gspread                   5.1.1                    pypi_0    pypi
gspread-formatting        1.0.5                    pypi_0    pypi
httplib2                  0.20.4                   pypi_0    pypi
idna                      3.3                      pypi_0    pypi
imagecorruptions          1.1.2                    pypi_0    pypi
imagehash                 4.2.1                    pypi_0    pypi
imageio                   2.9.0                    pypi_0    pypi
imgaug                    0.4.0                    pypi_0    pypi
importlib-resources       5.7.1                    pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561  
ipykernel                 6.13.0           py38h7f3c49e_0    conda-forge
ipython                   8.2.0            py38h06a4308_0  
ipython-genutils          0.2.0                    pypi_0    pypi
jedi                      0.18.1           py38h06a4308_1  
jinja2                    3.1.1                    pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
jsonschema                4.4.0                    pypi_0    pypi
jupyter_client            7.2.2            py38h06a4308_0  
jupyter_core              4.9.2            py38h06a4308_0  
jupyterlab-pygments       0.2.2                    pypi_0    pypi
kiwisolver                1.4.2                    pypi_0    pypi
labelbox                  2.7.0                    pypi_0    pypi
ld_impl_linux-64          2.35.1               h7274673_9  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0              h1d223b6_16    conda-forge
libgomp                   11.2.0              h1d223b6_16    conda-forge
libllvm11                 11.1.0               h3826bc1_1  
libsodium                 1.0.18               h7b6447c_0  
libstdcxx-ng              11.2.0              he4da1e4_16    conda-forge
llvmlite                  0.38.0           py38h4ff587b_0  
markupsafe                2.1.1                    pypi_0    pypi
matplotlib                3.3.4                    pypi_0    pypi
matplotlib-inline         0.1.2              pyhd3eb1b0_2  
mistune                   0.8.4                    pypi_0    pypi
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0            py38h7f8727e_0  
mkl_fft                   1.3.1            py38hd3c417c_0  
mkl_random                1.2.2            py38h51133e4_0  
mpire                     2.3.5                    pypi_0    pypi
nbclient                  0.6.0                    pypi_0    pypi
nbconvert                 6.5.0                    pypi_0    pypi
nbformat                  5.3.0                    pypi_0    pypi
ncurses                   6.3                  h7f8727e_2  
ndjson                    0.3.1                    pypi_0    pypi
nest-asyncio              1.5.5            py38h06a4308_0  
networkx                  2.8                      pypi_0    pypi
notebook                  6.4.11                   pypi_0    pypi
numba                     0.55.1           py38h4bf6c61_0    conda-forge
numpy                     1.21.5           py38he7a7128_1  
numpy-base                1.21.5           py38hf524024_1  
oauth2client              4.1.3                    pypi_0    pypi
oauthlib                  3.2.0                    pypi_0    pypi
opencv-python             4.5.5.64                 pypi_0    pypi
openpyxl                  3.0.9                    pypi_0    pypi
openssl                   1.1.1n               h7f8727e_0  
packaging                 21.3               pyhd3eb1b0_0  
pandas                    1.3.4                    pypi_0    pypi
pandocfilters             1.5.0                    pypi_0    pypi
parso                     0.8.3              pyhd3eb1b0_0  
pexpect                   4.8.0              pyhd3eb1b0_3  
pickleshare               0.7.5           pyhd3eb1b0_1003  
pillow                    8.1.0                    pypi_0    pypi
pip                       22.0.4             pyhd8ed1ab_0    conda-forge
prometheus-client         0.14.1                   pypi_0    pypi
prompt-toolkit            3.0.20             pyhd3eb1b0_0  
protobuf                  3.20.1                   pypi_0    pypi
psutil                    5.8.0            py38h27cfd23_1  
psycopg2-binary           2.9.3                    pypi_0    pypi
ptyprocess                0.7.0              pyhd3eb1b0_2  
pure_eval                 0.2.2              pyhd3eb1b0_0  
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pydantic                  1.9.0                    pypi_0    pypi
pygments                  2.11.2             pyhd3eb1b0_0  
pyparsing                 3.0.4              pyhd3eb1b0_0  
pyrsistent                0.18.1                   pypi_0    pypi
python                    3.8.13               h12debd9_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
python_abi                3.8                      2_cp38    conda-forge
pyturbojpeg               1.6.6                    pypi_0    pypi
pytz                      2022.1                   pypi_0    pypi
pywavelets                1.3.0                    pypi_0    pypi
pyzmq                     22.3.0           py38h295c915_2  
readline                  8.1.2                h7f8727e_1  
requests                  2.27.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.8                      pypi_0    pypi
scikit-image              0.19.2                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scipy                     1.8.0                    pypi_0    pypi
send2trash                1.8.0                    pypi_0    pypi
setuptools                62.1.0                   pypi_0    pypi
shapely                   1.8.1.post1              pypi_0    pypi
six                       1.16.0             pyhd3eb1b0_1  
sklearn                   0.0                      pypi_0    pypi
soupsieve                 2.3.2.post1              pypi_0    pypi
sqlite                    3.38.2               hc218d9a_0  
stack_data                0.2.0              pyhd3eb1b0_0  
tabulate                  0.8.7                    pypi_0    pypi
terminado                 0.13.3                   pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2022.4.22                pypi_0    pypi
tinycss2                  1.1.1                    pypi_0    pypi
tk                        8.6.11               h1ccaba5_0  
tornado                   6.1              py38h27cfd23_0  
tqdm                      4.59.0                   pypi_0    pypi
traitlets                 5.1.1              pyhd3eb1b0_0  
typing-extensions         4.2.0                    pypi_0    pypi
uritemplate               4.1.1                    pypi_0    pypi
urllib3                   1.26.9                   pypi_0    pypi
wcag-contrast-ratio       0.9                      pypi_0    pypi
wcwidth                   0.2.5              pyhd3eb1b0_0  
webcolors                 1.11.1                   pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0  
xlrd                      1.2.0                    pypi_0    pypi
xlsxwriter                1.3.7                    pypi_0    pypi
xz                        5.2.5                h7b6447c_0  
zeromq                    4.3.4                h2531618_0  
zipp                      3.8.0                    pypi_0    pypi
zlib                      1.2.12               h7f8727e_2

**Comment by rgommers** [comment]
The `cupyx.scipy.fft` question is answered; the install issue is probably resolved now that `11.0` is out for a while. Close this issue?
